packages/kit/src/runtime/client/renderer.js

```diff
@@ -84,11 +84,10 @@ export class Renderer {
 
 		/** @param {MouseEvent} event */
 		const trigger_prefetch = (event) => {
-			/** @type {HTMLAnchorElement | SVGAElement} */
+			if (!(event.target instanceof Node)) return;
 			const a = find_anchor(event.target);
-
 			if (a && a.hasAttribute('sveltekit:prefetch')) {
-				this.prefetch(new URL(a.href));
+				this.prefetch(new URL(a.href.toString()));
```

Comment:
ditto here. the addition of `.toString()` isn't going to break anyone's JS budget, but I think it's a useful and important principle that type safety shouldn't negatively impact end users

Reply:
I've never used svg. Are are sure this can't be a `SVGAnimatedString` as TypeScript suggests?

Reply:
It can, but arguments to `new URL` are automatically coerced, we don't need to pre-coerce them